### PR TITLE
[Docs] Fix some errors in `config.md` and `how_to.md`

### PR DESCRIPTION
**Pull request by kitecats**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Document error modification，change `--cfg-options data.train.pipeline.0.type=LoadImageFromNDArray`to` --cfg-options data.train.pipeline.0.type=LoadImageFromWebcam`.

### CODE REVIEW

**Comment by hhaAndroid** [comment]
@kitecats Thank you very much for your contribution. What is the error message, please

**Comment by kitecats** [comment]
This error occur in `mmyolo/docs/zh_cn/user_guides/config.md` 
TWF@M%AA@9N$PL5{Y$N2OEL

The original description is change `LoadImageFromFile` to `LoadImageFromWebcam`，but it was written incorrectly the `LoadImageFromNDArray`

File: docs/zh_cn/user_guides/config.md [hunk 1]
```diff
@@ -526,7 +526,7 @@ pre_transform = _base_.pre_transform # 变量 pre_transform 等于 _base_ 中定
 
 - 更新配置列表中的键
 
-  在配置文件里，一些字典型的配置被包含在列表中。例如，数据训练流程 `data.train.pipeline` 通常是一个列表，比如 `[dict(type='LoadImageFromFile'), ...]`。如果需要将 `'LoadImageFromFile'` 改成 `'LoadImageFromWebcam'`，需要写成下述形式：`--cfg-options data.train.pipeline.0.type=LoadImageFromNDArray`.
+  在配置文件里，一些字典型的配置被包含在列表中。例如，数据训练流程 `data.train.pipeline` 通常是一个列表，比如 `[dict(type='LoadImageFromFile'), ...]`。如果需要将 `'LoadImageFromFile'` 改成 `'LoadImageFromWebcam'`，需要写成下述形式：`--cfg-options data.train.pipeline.0.type=LoadImageFromWebcam`.
```
Comment (hhaAndroid) [inline docs/zh_cn/user_guides/config.md:529]:
```suggestion
  在配置文件里，一些字典型的配置被包含在列表中。例如，数据训练流程 `data.train.pipeline` 通常是一个列表，比如 `[dict(type='LoadImageFromFile'), ...]`。如果需要将 `'LoadImageFromFile'` 改成 `'LoadImageFromNDArray'`，需要写成下述形式：`--cfg-options data.train.pipeline.0.type=LoadImageFromNDArray`.
```

**Comment by hhaAndroid** [comment]
@kitecats Thank you。 I have commented

File: docs/zh_cn/advanced_guides/how_to.md [hunk 2]
```diff
@@ -38,7 +40,7 @@ model = dict(
             deepen_factor=deepen_factor,
             widen_factor=widen_factor,
             in_channels=[256, 512, 1024],
-            out_channels=[256, 512, 1024],
+            out_channels=[256, 512, 1024], # 因为受到widen_factor的影响，YOLOv5PAFPN的的out_channel=out_channel*widen_factor
```
Comment (RangeKing) [inline docs/zh_cn/advanced_guides/how_to.md:43]:
```suggestion
            out_channels=[256, 512, 1024], # 因为 out_channels 由 widen_factor 控制，YOLOv5PAFPN 的 out_channels = out_channels * widen_factor
```

File: docs/zh_cn/advanced_guides/how_to.md [hunk 3]
```diff
@@ -55,4 +57,6 @@ model = dict(
             # disable zero_init_offset to follow official implementation
             zero_init_offset=False)
     ]
+    bbox_head=dict(head_module=dict(in_channels=[512,512,512])) # 因为 out_channels 由 widen_factor 控制，YOLOv5HeadModuled 的 in_channels * widen_factor 才会等于最后一个 neck 的 out_channlels
```
Comment (hhaAndroid) [inline docs/zh_cn/advanced_guides/how_to.md:60]:
```suggestion
    bbox_head=dict(head_module=dict(in_channels=[512,512,512])) # 因为 out_channels 由 widen_factor 控制，YOLOv5HeadModuled 的 in_channels * widen_factor 才会等于最后一个 neck 的 out_channels
```

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**